eras/conway/impl/src/Cardano/Ledger/Conway/TxBody.hs

```diff
@@ -512,7 +513,7 @@ instance Crypto c => ConwayEraTxBody (ConwayEra c) where
     lensMemoRawType ctbrVotingProcedures (\txb x -> txb {ctbrVotingProcedures = x})
   {-# INLINE votingProceduresTxBodyL #-}
   proposalProceduresTxBodyL =
-    lensMemoRawType ctbrProposalProcedures (\txb x -> txb {ctbrProposalProcedures = x})
+    lensMemoRawType (SOS.toStrictSeq . ctbrProposalProcedures) (\txb x -> txb {ctbrProposalProcedures = SOS.fromStrictSeq x})
```

Comment:
Let's change proposals to `OSet` in a public interface, since this is a new field not present in previous eras, we are not introducing any breakage.
```suggestion
    lensMemoRawType (ctbrProposalProcedures) (\txb x -> txb {ctbrProposalProcedures = x})
```